### ISSUE TITLE
 error converting YAML to JSON: yaml

### ISSUE THREAD

**Issue by syncharny** [codecentric/helm-charts#267]
I get this error when trying to deploy this chart:

`Error: YAML parse error on keycloak/templates/statefulset.yaml: error converting YAML to JSON: yaml: line 90: did not find expected '-' indicator`

My config is as follows:

```
extraContainers: |
   - name: cloudsql-proxy
    image: gcr.io/cloudsql-docker/gce-proxy:1.17
    command:
      - /cloud_sql_proxy
    args:
      - -instances={{ .Values.cloudsql.project }}:{{ .Values.cloudsql.region }}:{{ .Values.cloudsql.instance }}=tcp:5432
      - -credential_file=/secrets/cloudsql/credentials.json
    volumeMounts:
      - name: cloudsql-creds
        mountPath: /secrets/cloudsql
        readOnly: true

extraVolumes: |
  - name: cloudsql-creds
    secret:
      secretName: postgres-sa

extraEnv: |
  - name: DB_VENDOR
    value: postgres
  - name: DB_ADDR
    value: 127.0.0.1
  - name: DB_PORT
    value: 5432
  - name: DB_DATABASE
    value: keycloak
  - name: DB_USER_FILE
    value: /secrets/db-creds/user
  - name: DB_PASSWORD_FILE
    value: /secrets/db-creds/password

extraVolumeMounts: |
  - name: db-creds
    mountPath: /secrets/db-creds
    readOnly: true

extraVolumes: |
  - name: db-creds
    secret:
      secretName: sql-keycloak-userpass
```

**Comment by cagri-rf** [comment]
It's related to indentation problem on the documentation, it should looks like below: 

```
extraContainers: |
   - name: cloudsql-proxy
     image: gcr.io/cloudsql-docker/gce-proxy:1.17
     command:
       - /cloud_sql_proxy
     args:
       - -instances={{ .Values.cloudsql.project }}:{{ .Values.cloudsql.region }}:{{ .Values.cloudsql.instance }}=tcp:5432
       - -credential_file=/secrets/cloudsql/credentials.json
     volumeMounts:
       - name: cloudsql-creds
         mountPath: /secrets/cloudsql
        readOnly: true
```

**Comment by unguiculus** [comment]
Indentation fixed in #289. Thanks for spotting that.
